lib/utilities/relationship-utils.js

```diff
@@ -69,3 +71,18 @@ function getIncludesArray (query) {
 function shouldIncludeRelationships (method) {
   return method.toLowerCase() === 'get';
 }
+
+function isJSONAPIPagination (req) {
+  return _.isPlainObject(req.query) && req.query.hasOwnProperty('page') &&
+    req.query['page'].hasOwnProperty('size');
+}
+
+/**
+ * Get the pagination data from query parameters
+ */
+function getPagination (query) {
+  return {
+    limit: query.page.size,
+    skip: Math.max(query.page.number - 1, 0) * query.page.size
+  };
```

Comment:
@taras, agree on testing - i'll get to it during the week.
@digitalsadhu, as I understand supporting it should involve forwarding loopback pagination params into ctx.filter as is - 
from there loopback will take charge.
should be straightforward (correct me if i'm wrong)